pkg/neorpc/result/version.go

```diff
@@ -65,15 +65,18 @@ type (
 		ValidatorsHistory map[uint32]uint32
 	}
 
-	// Application represents the node configuration.
+	// Application represents the node application configuration.
+	// Application is a NeoGo-specific extension to the node that is
+	// returned by the server if it is enabled.
 	Application struct {
-		// Below are NeoGo-specific extensions to the node that are
-		// returned by the server in case they're enabled.
-
```

Comment:
It's not a part of 5393e096d04200449d1bf06f19ce05585a277435. It should be a part of f8bebcd5a83c313cc09e8c8074686770b2889ab2.